src/main/scala/jtag/JtagTap.scala

```diff
@@ -31,7 +31,8 @@ class JtagOutput(irLength: Int) extends Bundle {
 }
 
 class JtagControl extends Bundle {
-  val jtag_reset = Input(Bool())
+  val jtag_reset = Input(Reset())
```

Comment:
still curious why this is not `AsyncReset` and why the `tapReset` is `Bool` vs `AsyncReset`

Reply:
Done.